Dockerfile

```
@@ -1,7 +1,54 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0 AS builder
+
+ARG APP_VERSION=0.0.0
+
+COPY . /app
+
+RUN cd /app && mkdir /app/bin \
+    && dotnet publish src/CycloneDX.BomRepoServer/CycloneDX.BomRepoServer.csproj \
+      --nologo \
+      --configuration Release \
+      --output bin \
+      --no-self-contained \
+      -p:Version=${APP_VERSION}
+
 FROM mcr.microsoft.com/dotnet/aspnet:6.0
 
-COPY bin/ /cyclonedx
+ENV TZ=Etc/UTC \
+    LANG=C.UTF-8 \
+    REPO__DIRECTORY=Repo \
```

```suggestion
    REPO__DIRECTORY=/repo \
```

I would like to propose changing the default REPO_DIRECTORY for the container to a actual path that is also used within the usage sections and makes sense over the 'default' value that would be assumed for `REPO__DIRECTORY` is just a relative path.

---

That change makes sense. I'll have to remember to communicate it though as it could be breaking for some folks with this already deployed.

---

While this sure can have breaking effects, I would guess the default Port change to 8080 would the more significant breaking change :thinking: